### PR TITLE
Astro 11/8 release notes

### PR DESCRIPTION
**Pull request by jwitz**

### CODE REVIEW

File: astro/release-notes.md [hunk 1]
```diff
@@ -1,3 +1,5 @@
+
+
```
Comment (jwitz) [inline astro/release-notes.md:2]:
```suggestion
```

File: astro/release-notes.md [hunk 2]
```diff
@@ -17,6 +17,41 @@ If you have any questions or a bug to report, reach out to [Astronomer support](
 
 **Latest CLI Version**: 1.6.1 ([Release notes](cli/release-notes.md))
 
+## November 8, 2022
+
+### New DAG deployment method
+
+Using Astro CLI 1.7, you can run `astro deploy -—dags` to push only the `dags` directory of your Astro project to a Deployment on Astro. This is an alternative to rebuilding your Astro project's Docker image every time you change your code.
```
Comment (kwcanuck) [inline astro/release-notes.md:24]:
```suggestion
In Astro CLI 1.7, you can run `astro deploy -—dags` to push only the `dags` directory of your Astro project to a Deployment on Astro. This is an alternative to rebuilding your Astro project's Docker image every time you change your code.
```

File: astro/release-notes.md [hunk 3]
```diff
@@ -17,6 +17,41 @@ If you have any questions or a bug to report, reach out to [Astronomer support](
...
+
+DAG-only deploys:
+
+- Are significantly faster than running `astro deploy` to rebuild your project.
+- Don't cause your workers to terminate and restart every time you need to make a change to a DAG, which leads to performance and cost benefits.
```
Comment (kwcanuck) [inline astro/release-notes.md:29]:
```suggestion
- Improve performance and reduce costs by removing the requirement to terminate and restart your workers every time you a change a DAG.
```

File: astro/release-notes.md [hunk 4]
```diff
@@ -17,6 +17,41 @@ If you have any questions or a bug to report, reach out to [Astronomer support](
...
 
+## November 8, 2022
+
+### New DAG deployment method
+
+Using Astro CLI 1.7, you can run `astro deploy -—dags` to push only the `dags` directory of your Astro project to a Deployment on Astro. This is an alternative to rebuilding your Astro project's Docker image every time you change your code.
+
+DAG-only deploys:
+
+- Are significantly faster than running `astro deploy` to rebuild your project.
```
Comment (kwcanuck) [inline astro/release-notes.md:28]:
```suggestion
- Speed project rebuilds significantly over running a full `astro deploy`.
```

File: astro/release-notes.md [hunk 5]
```diff
@@ -17,6 +17,41 @@ If you have any questions or a bug to report, reach out to [Astronomer support](
...
+- Don't cause your workers to terminate and restart every time you need to make a change to a DAG, which leads to performance and cost benefits.
+- Enable teams to create separate CI/CD processes for deploying DAGs and deploying other changes to your Astro project. See [Two-step templates](ci-cd.md#two-step-templates) for examples.
```
Comment (kwcanuck) [inline astro/release-notes.md:30]:
```suggestion
- Allow the creation of separate CI/CD processes for DAG deploys and changes to Astro projects. See [Two-step templates](ci-cd.md#two-step-templates) for examples. 
```

File: astro/release-notes.md [hunk 6]
```diff
@@ -17,6 +17,41 @@ If you have any questions or a bug to report, reach out to [Astronomer support](
...
+
+- Are significantly faster than running `astro deploy` to rebuild your project.
+- Don't cause your workers to terminate and restart every time you need to make a change to a DAG, which leads to performance and cost benefits.
+- Enable teams to create separate CI/CD processes for deploying DAGs and deploying other changes to your Astro project. See [Two-step templates](ci-cd.md#two-step-templates) for examples. 
+
+For more information, see [DAG-only deploys](deploy-code.md#dag-only-deploys).
+
+### Improved data lineage interface
+
+The **Lineage** tab in the Cloud UI has been updated with more features and now matches the look and feel of Astro.
```
Comment (kwcanuck) [inline astro/release-notes.md:36]:
```suggestion
New features have been added to the **Lineage** page in the Cloud UI and it now matches the appearance of Astro.
```
How does the page match the appearance of Astro? Do you mean other pages in the Cloud UI? Maybe:

The appearance and functionality of the *Lineage** page in the Cloud UI are now improved.

File: astro/release-notes.md [hunk 7]
```diff
@@ -17,6 +17,41 @@ If you have any questions or a bug to report, reach out to [Astronomer support](
...
+
+![Updated lineage page](/img/release-notes/lineage-integrated.png)
+
+Specifically, the tab includes the following improvements:
+
+- The process for comparing runs uses a simpler interface and provides more information about the runs you're comparing. See [Compare lineage graphs from previous runs](data-lineage.md#compare-lineage-graphs-from-previous-runs).
+- Names for UI elements have been updated to more clearly represent Airflow resources. For example, **jobs** was renamed to **runs**, and the **Explore** tab was renamed to **Runs**.
```
Comment (kwcanuck) [inline astro/release-notes.md:43]:
```suggestion
- Names for UI elements have been updated to more clearly represent Airflow resources. For example, **jobs** is now **runs**, and the **Explore** tab is now **Runs**.
```

**Review by kwcanuck — APPROVED** [review]

File: astro/release-notes.md [hunk 8]
```diff
@@ -17,6 +17,41 @@ If you have any questions or a bug to report, reach out to [Astronomer support](
...
+
+The **Lineage** tab has new features and is better integrated into the Cloud UI.
+
+![Updated lineage page](/img/release-notes/lineage-integrated.png)
+
+Specifically, the tab includes the following improvements:
+
+- The process for comparing runs uses a simpler interface and provides more information about the runs you're comparing. See [Compare lineage graphs from previous runs](data-lineage.md#compare-lineage-graphs-from-previous-runs).
+- Names for UI elements have been updated to more clearly represent Airflow resources. For example, **jobs** is now **runs**, and the **Explore** tab is now **Runs**.
+- Lineage graphs include new colors and animations to show the flow of data as it moves between runs and datasets.
```
Comment (jwitz) [inline astro/release-notes.md:44]:
```suggestion
```
Deleting for now until it's confirmed in production

File: astro/release-notes.md [hunk 9]
```diff
@@ -17,6 +17,30 @@ If you have any questions or a bug to report, reach out to [Astronomer support](
...
+
+DAG-only deploys:
+
+- Are significantly faster than running `astro deploy` to rebuild your project.
+- Improve performance and reduce costs by removing the requirement to terminate and restart your workers every time you a change a DAG.
+- Allow the creation of separate CI/CD processes for DAG deploys and changes to Astro projects. See [CI/CD templates](ci-cd.md#ci-cd-templates) for examples. 
+
+For more information, see [DAG-only deploys](deploy-code.md#dag-only-deploys).
+
+
```
Comment (jwitz) [inline astro/release-notes.md:34]:
```suggestion
```

File: astro/release-notes.md [hunk 10]
```diff
@@ -17,6 +17,30 @@ If you have any questions or a bug to report, reach out to [Astronomer support](
...
+Using Astro CLI 1.7, you can run `astro deploy -—dags` to push only the `dags` directory of your Astro project to a Deployment on Astro. This is an alternative to rebuilding your Astro project's Docker image every time you change your code.
+
+DAG-only deploys:
+
+- Are significantly faster than running `astro deploy` to rebuild your project.
+- Improve performance and reduce costs by removing the requirement to terminate and restart your workers every time you a change a DAG.
+- Allow the creation of separate CI/CD processes for DAG deploys and changes to Astro projects. See [CI/CD templates](ci-cd.md#ci-cd-templates) for examples. 
+
+For more information, see [DAG-only deploys](deploy-code.md#dag-only-deploys).
```
Comment (jwitz) [inline astro/release-notes.md:32]:
```suggestion
For more information, see [Deploy DAGs only](deploy-code.md#deploy-dags-only).
```

**Pull request merged**